app/uk/gov/hmrc/individualsemploymentsapi/controller/v2/CommonControllerV2.scala

```diff
@@ -14,27 +14,24 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.individualsemploymentsapi.controller
+package uk.gov.hmrc.individualsemploymentsapi.controller.v2
 
-import javax.inject.Inject
 import org.joda.time.DateTime
 import play.api.Logger
 import play.api.mvc.{ControllerComponents, Request, RequestHeader, Result}
+import uk.gov.hmrc.auth.core.{AuthorisationException, AuthorisedFunctions, Enrolment, InsufficientEnrolments}
 import uk.gov.hmrc.auth.core.authorise.Predicate
 import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals
-import uk.gov.hmrc.auth.core.{AuthorisationException, AuthorisedFunctions, Enrolment, InsufficientEnrolments}
 import uk.gov.hmrc.http.{BadRequestException, HeaderCarrier, InternalServerException, TooManyRequestException}
 import uk.gov.hmrc.individualsemploymentsapi.audit.v2.AuditHelper
-import uk.gov.hmrc.individualsemploymentsapi.controller.Environment.SANDBOX
-import uk.gov.hmrc.individualsemploymentsapi.error.ErrorResponses._
-import uk.gov.hmrc.individualsemploymentsapi.util.Dates._
+import uk.gov.hmrc.individualsemploymentsapi.error.ErrorResponses.{ErrorInternalServer, ErrorInvalidRequest, ErrorNotFound, ErrorTooManyRequests, ErrorUnauthorized, MatchNotFoundException, MissingQueryParameterException}
+import uk.gov.hmrc.individualsemploymentsapi.util.Dates.toFormattedLocalDate
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
-import scala.xml.dtd.ValidationException
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
 
-abstract class CommonController @Inject()(cc: ControllerComponents) extends BackendController(cc) {
+abstract class CommonControllerV2 @Inject()(cc: ControllerComponents) extends BackendController(cc) {
```

Comment:
Can we please remove the version from the file name and class name as its already described in the package?